CONTRIBUTING.no.md

```diff
@@ -0,0 +1,154 @@
+
+- [AGC utskrift for Comanche][8]
+- [AGC utskrift for Luminary][9]
+
+## Nytteprogrammer
+
+GitHub har kildekode støtte for «assembly» programmerings språket AGC ( innebygget ), og de følgene tekst redigerings programmene har også støtte for nøkkel ord fargesetting:
+
+- [Atom][Atom]†
+- [CodeBlocks][CodeBlocks]
+- [Eclipse][Eclipse]
+- [Kate][Kate]
+- [ProgrammersNotepad][ProgrammersNotepad]
+- [Sublime Text 3][Sublime Text]†
+- [TextPad][TextPad]
+- [Vim][Vim]
+- [Visual Studio Code][VisualStudioCode]†
+- [jEdit][jEdit]
+
+† Supporterer automatisk formattering
+
+[Atom]:https://github.com/Alhadis/language-agc
+[CodeBlocks]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/CodeBlocks
+[Eclipse]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/Eclipse
+[Kate]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/Kate
+[ProgrammersNotepad]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/ProgrammersNotepad
+[Sublime Text]:https://github.com/jimlawton/AGC-Assembly
+[TextPad]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/TextPad
+[Vim]:https://github.com/wsdjeg/vim-assembly
+[VisualStudioCode]:https://github.com/wopian/agc-assembly
+[jEdit]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/jEdit
+
+## Formattering
+
+**Merk:** GitHub og redigerings programmene nevnt ovenfor vil sørge for at du bruker korrekt formattering av kilde koden.
+
+-	Bruk Tab for innrykk
+-	Bruk Tab med 8 mellomrom
+-	Ta bort mellomrom og annet som ikke er synlig på slutten av setninger
+
+## Hva må jeg sjekke?
+
+Eventuelle avvik ifra de innskannede bildene og kilde koden i dette «repository»
+
+### Kommentarer
+
+Kommentarer i denne koden **SKAL** stemme **eksakt** med den digitaliserte originalen.
+
+Vanlige feil en bør se etter er blant annet:
+
+#### Typografiske feil
+
+Noen steder skrev original utvikleren feil i kode kommentarene. Noen av disse feilene ble rettet under digitaliseringen, og digitaliseringen har også introdusert nye typografiske feil som ikke kommer ifra papir utskriften.
+
+For eksempel, om de originale dokumentene var feilstavet til ‘SPAECRAFT’ og de digitaliserte dokumentene ble endret til ‘SPACECRAFT’ så **SKAL** dette rettes til og reflektere de originale dokumentene, altså uten ‘C’ i ordet.
+
+På samme måte, om digitaliseringen introduserte stavefeil som ikke var tilstede i de originale dokumentene **SKAL** dette rettes.
+
+#### Mellomrom
+
+- Mellomrom imellom to karakterer i teksten skal respektere følgende regler ( se diskusjon i #316 )
+  - Enkelt mellomrom for nye ord
+  - Dobbelt mellomrom for nye linjer
+  - Trippel mellomrom for innrykk
+
+For eksempel skal følgende kode:
+
+```plain
+	1)  FOO BAR BAZ QUX QUUX QUUZ. CORGE, GRAULT,
+	GARPLY, WALDO.
+```
+
+Se slik ut:
+
+```plain
+	1) FOO BAR BAZ QUX QUUX QUUZ.  CORGE, GRAULT,
+	   GARPLY, WALDO.
+```
```

Comment:
This section (lines 100-112, including example that was causing confusion) was replaced with:

> Not all pages in the scans follow this generalisation, if the scans only have a single space instead of a double space, use a single space.